server/app/views/v1/grids/_grid.json.jbuilder

```diff
@@ -1,6 +1,5 @@
 json.id grid.to_path
 json.name grid.name
-json.token grid.token
```

Comment:
:-1: can't break the existing API in the same PR, this needs to be done much later, once all the plugins have been updated.